.github/workflows/long-test.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request:  # TODO remove this before merge
```

Comment:
Add a workflow dispatch, and a label trigger.

Reply:
> and a label trigger

Do we have an example already? I don't see workflow-specific label triggers in the doc, only job-specific.

Reply:
No, but I think we use https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Annoyingly I think we need extra if conditions to check the value of the label, which means a bit of running on every PR effectively, and a situation where it's harder to tell if you really have triggered the LTCI or not. Ugh.

Reply:
This is what I mean, we have to copy-paste the condition into each job, and will be green if all checks disabled, which is confusing.

Is it better to manually start it from Actions if needed instead?

Reply:
> Is it better to manually start it from Actions if needed instead?

Not as far as I can tell. The alternative is the canary file I think. Maybe we can still do the condition, but set a check status that does not lead to green?

Reply:
Manual start via workflow_dispatch only works on branches, not forks. Maybe that is an acceptable restriction though.